plugin/kermit-ir-plugin/src/test/kotlin/co/touchlab/kermit/irplugin/KermitStripperIrPluginTest.kt

```diff
@@ -8,16 +8,19 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+@file:OptIn(ExperimentalCompilerApi::class)
```

Comment:
Should we declare `optIn` at `build.gradle.kts` level, so we don't have to add to individual files?

Reply:
Like this,
https://github.com/touchlab/KaMPKit/blob/70be2d8ec093aea3511d0f86258faed1d6c3fcf9/shared/build.gradle.kts#L42